proto/src/determined/api/v1/model.proto

```diff
@@ -101,8 +101,10 @@ message PostModelResponse {
 
 // Request for updating a model in the registry.
 message PatchModelRequest {
+  // The id of the model being updated.
+  int32 model_id = 1;
```

Comment:
is this just to make the documentation look "good"?

Reply:
I will go un-change the proto numbering so we aren't confusing previous versions of these requests.

We do actually need model_id here and in PatchModelVersionRequest because (my current understanding) that's where we get the input from the `/models/{model_id}` url into PatchModelRequest, and not in a subfield of the Model